src/renderers/shared/fiber/ReactFiberHostContext.js

```diff
@@ -80,26 +89,13 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(
   }
 
   function getHostContext(): CX {
-    const context = contextStackCursor.current;
-    invariant(
-      context != null,
-      'Expected host context to exist. This error is likely caused by a bug ' +
-        'in React. Please file an issue.',
-    );
+    const context = requiredContext(contextStackCursor.current);
     return context;
   }
 
   function pushHostContext(fiber: Fiber): void {
-    const rootInstance = rootInstanceStackCursor.current;
-    invariant(
-      rootInstance != null,
-      'Expected root host context to exist. This error is likely caused by ' +
-        'a bug in React. Please file an issue.',
-    );
-
-    const context = contextStackCursor.current !== null
-      ? contextStackCursor.current
-      : emptyObject;
+    const rootInstance = requiredContext(rootInstanceStackCursor.current);
+    const context = requiredContext(contextStackCursor.current);
```

Comment:
At the very least, it was inconsistent with our declared type signatures for the host config (but emptyObject isn't typed so).